packages/api-clients/open-api/apiGatewayApi.yml

```diff
@@ -11,7 +11,7 @@ info:
   x-api-id: an x-api-id
   x-summary: an x-summary
 servers:
-  - url: "{{baseUrl}}/{{version}}"
+  - url: "/api-gateway/0.0"
```

Comment:
few consierations:
- version is `0.0` in dev and `1.0` in prod
- `baseUrl` is replaced with the actual external url